app/Repositories/User/UserRepository.php

```diff
@@ -75,4 +74,19 @@ public function parseToken($accessToken)
             return false;
         }
     }
+
+    /**
+     * @param $notifications
+     * @return array
+     */
+    public function fetchListing($notifications)
+    {
```

Comment:
Add some description of the function,  and perhaps, define this function in the interface as well.